src/styles/index.css

```diff
@@ -0,0 +1,70 @@
+.stopwatch__circle,
+.stopwatch__second,
+.stopwatch__min,
+.stopwatch,
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
The properties `display: flex;`, `justify-content: center;` and `align-items: center;` are duplicated for several classes. Consider creating a general utility class for centering elements and apply it to the necessary elements in your HTML. This will help to avoid code duplication and make your CSS more maintainable.